api/src/certification/course/domain/usecases/get-next-challenge-for-v2-certification.js

```diff
@@ -0,0 +1,13 @@
+const getNextChallengeForV2Certification = async function ({
+  assessment,
+  certificationChallengeRepository,
+  challengeRepository,
+}) {
+  return certificationChallengeRepository
+    .getNextNonAnsweredChallengeByCourseId(assessment.id, assessment.certificationCourseId)
+    .then((certificationChallenge) => {
+      return challengeRepository.get(certificationChallenge.challengeId);
+    });
```

Comment:
On sort un peu du cadre mais: 

```suggestion
  const certificationChallenge = await certificationChallengeRepository
    .getNextNonAnsweredChallengeByCourseId(assessment.id, assessment.certificationCourseId)
  return challengeRepository.get(certificationChallenge.challengeId);
```